salt/states/virt.py

```diff
@@ -355,6 +357,11 @@ def defined(
 
         .. deprecated:: sodium
 
+    :param serials: list of serial connections
+        .. versionadded:: Sodium
+    :param consoles: list of console parameters
+        .. versionadded:: Sodium
+
```

Comment:
Like for the `update()` documentation, add a link to the list items definition that you will add to `init()`